numpy/core/src/multiarray/nditer_api.c

```diff
@@ -1380,47 +1380,6 @@ NpyIter_GetInnerLoopSizePtr(NpyIter *iter)
     }
 }
 
-/*NUMPY_API
- * Resolves all writebackifcopy scratch buffers, not safe to use iterator
- * operands after this call, in this iterator as well as any copies.
- * Returns 0 on success, -1 on failure
- */
-NPY_NO_EXPORT int
-NpyIter_Close(NpyIter *iter)
```

Comment:
Doesn't the code below need to move into `NpyIter_Dealloc`?

Reply:
Yes, it now lives inside `NpyIter_Deallc`. Comment expanded to emphasize that.